WordPress/Classes/ViewRelated/NUX/SiteCreationDomainsViewController.swift

```diff
@@ -0,0 +1,253 @@
+import UIKit
```

Comment:
The view rotates to landscape on iPhone. Maybe we should make it portrait only to be consistent?